src/modules/CRUD/components/DataTable/Footer/Footer.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactPaginate from 'react-paginate';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { parseUrl } from 'query-string';
+
+import {
+  Alignment,
+  Classes,
+  Label,
+  Navbar,
+  NavbarGroup,
+  HTMLSelect as Select,
+} from '@blueprintjs/core';
+
+import './styles.scss';
+
+const getPageFromUrl = string => {
+  const { query: { page } } = parseUrl(string);
+  return Number(page);
+};
+
+const getCurrentPage = (prevPage, nextPage) => {
+  if (prevPage === null && nextPage === null) {
+    return null;
+  }
+  return prevPage ? prevPage + 1 : nextPage - 1;
+};
+
+const Footer = ({
+  featuresList: { count = 0, previous = null, next = null } = {},
+  onPageChange,
+  pageSize,
+  querystring: { page },
+  setPageSize,
+  t,
+}) => {
+  const prevPage = previous ? getPageFromUrl(previous) : null;
+  const nextPage = next ? getPageFromUrl(next) : null;
+  const currentPage = getCurrentPage(prevPage, nextPage);
+
+  if (currentPage === null || !count) {
+    return null;
+  }
+
+  const forcePage = (page || currentPage) - 1;
+
+  const pageCount = Math.ceil(count / pageSize);
+  const pageRangeDisplayed = pageCount >= 4 ? 4 : pageCount;
```

Comment:
Is `4` a magic value? ;)
If it is, a comment could explain why.
Otherwise, this should be a named constant.